### ISSUE TITLE
Let's describe the slices of the corner guides & the number of side-length units

### ISSUE THREAD

**Issue by terracoda** [phetsims/quadrilateral#231]
Related to comparative size parameters defined in #219, I would like to add a described measurement or reference point for the size of the corners and sides. This approach will also make two of the visual scaffolds accessible to more learners including those who have vision impairments.

Here are the parameters and their mappings. 
### Table: Scale for describing slices and side lengths

Strings for Slices (Corner Angle Units) | Mappings for Slices | Strings for Side-length Units | Mappings for Units
------------------------------------------ | ------------------ | ----------------------------- | -------------------
{{less than half}} {{1 slice}} | < 15 degrees | {{less than half }} {{1 unit}} | < ½ a unit
{{half}} {{1 slice}} | = 15 degrees | {{about half }} {{1 unit}} | = ½ a unit
{{just under}} {{1 slice}} | > 15 & < 30 degrees | {{just under}} {{1 unit}} | > ½ a unit & < 1 unit
{{1 slice}} | = 30 degrees | {{1}} unit | = 1 unit
{{just over}} {{1 slice}} | > 30 & < 45 degrees | just over {{1}} unit | > 1 unit & < 1 ½ units
{{1}} {{and a half}} slices | = 45 degrees | {{1 }} {{and a half}} units | = 1 ½ a unit
{{just under}} {{2}} slices | > 45 & < 60 degrees | {{just under}} {{2}} units | > 1 ½ & < 2 units
{{2}} slices | = 60 degrees | {{2}} units | = 2 units
{{just over}} {{2}} slices | > 60 & < 75 degrees | {{just over}} {{2}} units | > 2 units & < 2 ½ units
{{2}} {{and a half}} slices | = 75 degrees | {{2}} {{and a half}} units | = 2 ½ a unit
{{just under}} {{3}} slices | > 75 & < 90 degrees | {{just under}} {{3}} units | > 2 ½ & < 3 units
{{right angle}} | = 90 degrees | {{3}} units | =  3 units
{{just over}} {{3}} slices | > 90 & < 105 degrees | {{just over}} {{3}} units | > 3 & < 3 ½ units
… | … | … | …
{{angle flat}} | = 180 degrees | Continue as above | Continue as above

The idea is to put this information before the corner and side descriptions like this:
- Corner A, **{{1 slice}},** {{far smaller than}} opposite corner, similar but wider than equal adjacent corners
- Corner B, **{{just under}} {{1 slice}},** {{equal to}} opposite corner, smaller than adjacent corners
- Corner C, **{{just over}} {{9}} slices,** far wider than opposite corner, far wider than equal adjacent corners
- Corner D, **{{just under}} {{1 slice}},** {{equal to}} opposite corner, smaller than adjacent corners
- Side AB, **{{just over}} {{6}} units,** {{about twice as long as}} opposite side, equal to one adjacent side
- Side BC, **{{just under}} {{3}} units,** {{about half as long as}} opposite side, equal to one adjacent side
- Side DA, **{{just over}} {{6}} units,** {{about twice as long as}} opposite side, equal to one adjacent side

For a shape like this Dart:
<img width="337" alt="image" src="https://user-images.githubusercontent.com/1902116/193296910-5dd2e63c-1aae-408b-a3d0-3713fb58b3ed.png">

**Comment by BLFiedler** [comment]
I may actually suggest another word than pie, or to just remove it altogether and stick with slice. Pie is a bit close to Math Pi, which is very relevant in this case. Math Pi is equal to half of the circle, 2*Pi the whole circle. One "pie" slice is Math Pi/6.

**Comment by terracoda** [comment]
Good point about Pi versus pie.

**Comment by terracoda** [comment]
I removed "pie", so now we just have "slice" and "slices".

**Comment by terracoda** [comment]
@jessegreenberg, big question for me for implementation is:
- For now, I want to add this additional string for the "initial" object response, for example when a learner clicks/taps on a corner or side, or moves focus to a corner or side, or presses Space or Enter on a focused corner or side.

I don't think the pies and slices will be used in changing object responses for corners and sides.

**Comment by terracoda** [comment]
And just a heads up, the "slice" descriptions might need to be limited to when the Corner Guides are checked.

**Comment by terracoda** [comment]
In yesterday's design meeting, we all agreed to just describe the corner "slices" when the Corner guides checkbox is checked. I am assuming the side "units" can be present all the time as they are always present.

@jessegreenberg, I think you can start implementing these strings now.

**Comment by jessegreenberg** [comment]
I had a question about the "right angle" that is currently included in the object response when corner guides are hidden. But now a similar description is being added when corner guides are shown. Right angle indicators are not shown unless corner guides are visible. Over slack:

> Currently, we still hear "right angle" when corner guides are not visible. Should we still hear "right angle" in this case or should it be removed?
> Code is slightly more simple if it is removed in this case. That could also make sense because right angle indicators are not shown anymore unless "Angles" checkbox is checked. But I can definitely do either way.

> Let's remove it. Right angles are mentioned in the Context Responses for the shapes that require them. And now that we have the option to describe the angles somewhat quantitatively, let's keep this description in the quantitative category.
> That is, let's keep this description in the quantitative category for Object Responses.

**Comment by jessegreenberg** [comment]
OK these are now in. Over slack I mentioned to @terracoda that the tolerances (interAngleToleranceInterval and shapeLengthToleranceInterval) make it difficult to find some of these descriptions like "1 and a half units" or "2 and a half slices". The descriptions are correct for the model though. In https://github.com/phetsims/quadrilateral/issues/219#issuecomment-1255192232 we made these easily findable by widening the ranges just for "half". But maybe none of that is necessary here, its easy to hear all of the length unit descriptions if you align with the grid.

Over to you to try it out @terracoda!

**Comment by terracoda** [comment]
I found  "1 and a half slices, half as wide as opposite corner" with fine input.
<img width="897" alt="image" src="https://user-images.githubusercontent.com/1902116/195606188-7da592a4-7de6-4e91-846e-5b86873898fb.png">

And I found "angle flat, half as wide as opposite corner" in this kite.
<img width="380" alt="image" src="https://user-images.githubusercontent.com/1902116/195605649-70e4a6a6-52a7-4ba7-a9f2-5478b72cdd07.png">

The goal is not to find 1.5 slices or or 2.5 units, but to have the ability to get a rough measurement of a side or corner.

I think it works really nicely with the keyboard as I can press Space or Enter to re-read the state of the side or corner, as I pause during investigations. I can repeat the object information with clicks in the same manner. Both seem really nice.

And for some shapes, where the equality of a corner might be ambiguous ("equal to one adjacent corner"), a learner can verify with voiced descriptions which corners are equal - very nice for trapezoids and kites.

When a learner has a convex or concave quad and they might be trying to make some other shape, they can verify a bit more easily which corner or size is not quite the right size or in the right place.

I think the slices and units are working great, so will close this issue.

I have a couple of more general design questions which I will open in a new issue for longer term consideration & impact.
